install.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/sh
-CSDIR="$HOME/.config/geany/colorschemes/"
-echo "Installing themes into \`$CSDIR'..."
+
+CSDIR="${XDG_CONFIG_HOME:-$HOME/.config}/geany/colorschemes/"
+echo "Installing themes into '$CSDIR'..."
 mkdir -p "$CSDIR"
-for SCHEME in `ls colorschemes/*.conf`
```

Comment:
BTW @codebrainz in case you wonder: using `ls` is not a great idea because the command substitution will split on spaces, whereas `for f in GLOB` will not, so it fixes support for filenames containing spaces -- not that it's great to have them, though.  Could also be solved by altering `IFS`, but it usually is a last recourse for special cases.  Also, I must admit that I fail to see any reason to call `ls` like that, was there one?  History suggest there isn't, but it's not clear.

Reply:
I never really figured anyone would use this script since it's a single command to install the thems (`cp colourschemes/* ~/.config/geany/colourschemes/`).

Reply:
The script is nice as it answers the question "how do I install" for someone new to the codebase. 

Also, by reading it, I learned that there is no magic.

Finally, even for someone who knows the codebase/an update bot, it is a nice wrapper.

Reply:
two commits - done